turborepo-vercel/package.json

```diff
@@ -8,7 +8,8 @@
     "build": "turbo run build",
     "dev": "turbo run dev --parallel",
     "format": "prettier --write .",
-    "lint": "turbo run lint"
+    "lint": "turbo run lint",
+    "typecheck": "turbo run typecheck"
```

Comment:
we're going to need to update `turbo.json` with this new job - it depends on build due to the ui package needing to been built so typescript can find it 

```json
"typecheck": {
  "dependsOn": ["^build"]
}
```